pkg/gui/commit_message_panel.go

```diff
@@ -12,16 +12,26 @@ func (gui *Gui) handleCommitConfirm(g *gocui.Gui, v *gocui.View) error {
 	if message == "" {
 		return gui.createErrorPanel(g, gui.Tr.SLocalize("CommitWithoutMessageErr"))
 	}
-	sub, err := gui.GitCommand.Commit(message)
-	if err != nil {
-		// TODO need to find a way to send through this error
-		if err != gui.Errors.ErrSubProcess {
-			return gui.createErrorPanel(g, err.Error())
+
+	c, isGPG := gui.GitCommand.CommitWithStatus(message)
+	if isGPG {
+		// put it into subprogress
+		sub, err := gui.GitCommand.Commit(message)
+		if err != nil {
+			// TODO need to find a way to send through this error
```

Comment:
is this TODO still necessary? using an error panel should be fine in my opinion

Reply:
I am not sure whether I can remove this, so I just keep it as before.